services/analytics-collector/pod-analytics-collector/src/storage.ts

```diff
@@ -0,0 +1,31 @@
+//
+// Copyright © 2024 Hardcore Engineering Inc.
+//
+// Licensed under the Eclipse Public License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may
+// obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import { Db, MongoClient } from 'mongodb'
+
+let client: MongoClient | undefined
+
+export const getDB = async (mongoUrl: string, db: string): Promise<Db> => {
+  client = new MongoClient(mongoUrl)
```

Comment:
Please use our MongoClient to properly pass MONGO_OPTIONS.

Reply:
Done.